src/cdk/schematics/update-tool/migration.ts

```diff
@@ -89,16 +85,9 @@ export class MigrationRule<T> {
     });
   }
 
-  /** Prints the specified message with "info" loglevel. */
+  /** Prints the given message with "info" loglevel. */
   printInfo(text: string) {
-    this.logger.info(`- ${this.tsconfigPath}: ${text}`);
+    // TODO: FIND REPLACEMENT???
```

Comment:
Elaborate on this TODO?

Reply:
Oh, I didn't mean to leave that in here. good catch, fixed.